sdk/communication/communication-chat/tsconfig.json

```diff
@@ -2,7 +2,10 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
-    "declarationDir": "./types"
+    "declarationDir": "./types",
+    "lib": [
+      "dom"
```

Comment:
We're supposed to avoid taking this dependency if we can since it forces consumers to do the same. Can you mock out `navigator` with a shim instead of doing this?

Example: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-lro/src/shims.d.ts